### DISCUSSION TITLE
Partition usage reached 100%

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#1929] [Unsupported Versions] [answered]
Hi Doug/Team,

Good Day everyone

We executed the Security Onion in our environment last Friday, Nov 13. However, after several days, we encountered an error stating "partition usage reached 100" this via wazuh email notification.

We tried to fix it by running so-elastic-clear to clean up the disk space. However, in running this command the kibana still don't work properly together with its services.

We tried to search any related issue in the community but still the same error. Please help us in this issue.

Thank you,
Dem

**Comment by weslambert** [comment, marked as answer]
Sounds like you chose an unlucky day to install 😉  (Fri 13) 🎃 .  If you reached the watermark threshold for Elasticsearch, you will need to run `so-elasticsearch-indices-rw` to reset the readonly attribute, and restart Elasticsearch with `so-elasticsearch-restart`.
